ddm_test_validate.py

```diff
@@ -0,0 +1,214 @@
+import logging
+import datetime
+import pathlib
+import os
+import time
+from distutils.util import strtobool
+from typing import Any, Dict, List
+
+import numpy as np
+import pandas as pd
+from azure.core.exceptions import HttpResponseError
+from dotenv import load_dotenv, set_key
+from microsoft_bonsai_api.simulator.client import BonsaiClient, BonsaiClientConfig
+from microsoft_bonsai_api.simulator.generated.models import (
+    SimulatorInterface,
+    SimulatorSessionResponse,
+    SimulatorState,
+)
+
```

Comment:
likewise, you aren't connecting to bonsai in this script so you don't need these imports

Reply:
Yes, I forgot to remove before. Will remove in the next commit.